src/app/reducers/environments.ts

```diff
@@ -17,16 +18,27 @@ export interface State {
 }
 
 export const getInitialEnvironmentState = (): EnvironmentState => {
+  const { initialData } = getAltairConfig();
   return {
     title: 'Environment',
-    variablesJson: '{}'
+    variablesJson: '{}',
+    id: uuid(),
```

Comment:
base environment doesn't need the `id`. Sorry about the confusion. Only sub environments do.